docs/CHANGELOG.md

```diff
@@ -14,50 +14,54 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ### Added
 
+- Add an audio subchapter to the pong chapter. ([#1842])
+
 ### Changed
 
 ### Fixed
 
+[#1842]: https://github.com/amethyst/amethyst/issues/1842
+
 ## [0.12.0] - 2019-07-30
 
 ### Breaking changes
 
-* `Float` newtype removed, moved back to `f32` primitive for all values ([#1747])
-* `TextureProcessor` and `MeshProcessor` systems are now separated from `RenderingSystem` ([#1772])
+- `Float` newtype removed, moved back to `f32` primitive for all values ([#1747])
+- `TextureProcessor` and `MeshProcessor` systems are now separated from `RenderingSystem` ([#1772])
 
 ### Added
 
-* Add a feature flag `sentry` to disable the sentry dependency. ([#1804]) ([#1825])
```

Comment:
Is updating the whole file intended?

Reply:
No, that was not intended. My editor (VS Code) automatically formats files whenever you save them. It's quite convenient most of the time, but it's mildly annoying for scenarios like this.

If it matters, I can temporarily disable the format-on-save feature and change the bullets back to `*`s instead of `-`s. Should I do it in a separate commit or should I amend the `Add changelog entry` commit and `git push --force`?

Reply:
If you ask to me, I'll do a push --force

Reply:
Fixed it! Resolving this conversation now.